ckanext/ontario_theme/fanstatic/common.css

```diff
@@ -29,4 +29,145 @@
 .toolbar .breadcrumb .active a,
 .toolbar .breadcrumb a.active {
   font-weight: normal;
+}
+/* ========================================================================
+   Page Alerts -Design System
+   ======================================================================== */
+   .ontario-alert{
+    border-width:0 0 0 0.9375rem;
+    border-style:solid;
+    padding:1.5rem;
+    margin:2rem 0 2.5rem 0;
+}
+.ontario-alert__header{
+    display:flex
+}
+.ontario-alert__header-icon{
+    display:inline-block;
+    vertical-align:top;
+    margin-right:.95rem;
+}
+.ontario-alert__header-icon svg{
+    width:36px;
+    height:36px
+}
+@media screen and (max-width: 40em){
+    .ontario-alert__header-icon svg{
+        width:32px;
+        height:32px
+    }
+}
+.ontario-alert__header-title{
+    margin-bottom:.75rem;
+    padding-top:2px
+}
+.ontario-alert__body{
```

Comment:
This class does not appear to be used. There is another class `.feedback.ontario-alert__body` on line 167 that seems to be the one being used.

Reply:
The icon we are currently using isn't from the DS so it creates some complications. Once we add a different icon, the `.feedback` class can be removed and the original DS page alert CSS can be used, without most of the overrides. I will add a comment.

Reply:
Ok let's leave it.